pyaqueduct/experiment.py

```diff
@@ -8,13 +8,16 @@
 from uuid import UUID
 
 from pydantic import BaseModel, Field, validate_call
+from typing_extensions import Annotated
 
 from pyaqueduct.client import AqueductClient
 
 _MAX_TITLE_LENGTH = 100
 _MAX_DESCRIPTION_LENGTH = 2000
 _MAX_TAG_LENGTH = 50
```

Comment:
Is this the right way/place to enforce the length? I think if database allows to limit the length of a data type, this should be enough to do in a database schema. Having another number in code may be a source of inconsistency.

Reply:
It is a trade-off between validating early or relying on the server to do that. At the time we implemented these constraints in PyAqueduct, there was no server-side validation, but now I see it obsolete and it is better to rely on the server validation. It is worth another issue to totally refactor that. I have created the issue here: https://github.com/AqueductHub/pyaqueduct/issues/24